api/VRPose.json

```diff
@@ -46,6 +46,10 @@
             "version_added": null
           },
           "safari": {
+            "version_added": "11.1",
```

Comment:
Can you separate out this change and just add safari_ios in this PR? I'm suspicious of this and would want to look closer at what else was shipped and unshipped along with it.